common/src/main/java/com/wynntils/features/user/overlays/CustomBarsOverlayFeature.java

```diff
@@ -183,6 +189,172 @@ public void render(PoseStack poseStack, float partialTicks, Window window) {
         }
     }
 
+    public static class AwakenedProgressBarOverlay extends Overlay {
+        @Config(key = "feature.wynntils.customBarsOverlay.overlay.awakenedProgressBar.textShadow")
```

Comment:
You only need to specify the key in special cases (where the auto generated one is not good, so when using inheritance). Remove them this case, only leave `@Config`

Reply:
Would I still need to add it to the language file? Or is the name just the name of the variable?

Reply:
Basically the key you defined here is already the default one. So yes, keep the translations, but no need to set the key here.

Reply:
ah a.